api-docs-slate/source/includes/_node.md

```diff
@@ -320,6 +459,59 @@ Parameter | Description
 tx | transaction hash
 
 
+## Estimate fee
```

Comment:
Okay, so this section depends on the result of https://github.com/bcoin-org/bcoin/pull/889

Reply:
However, it's likely nothing will need to change in this section.

Reply:
We can update if necessary.